### DISCUSSION TITLE
Reduce Ergo confirmations

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1487] [General]
hello... I found an answer from @oliverw  that said the following:

"pools": [
  {
    "id": "ergo",
    "paymentprocessing": {
      "minimumConfirmations": 70 
    }
  }
]

so... should my json stay like this? (check if the {  } placed are correct)

"pools": [
        // Repeat the following section for multiple coins
        {
            // DON'T change the id after a production pool has begun collecting shares!
            "id": "MYIDHERE",
			"paymentprocessing": {
			"minimumConfirmations": 70
			}	
            "enabled": true,
            "coin": "ergo",
            // Address to where block rewards are given (pool wallet)
            "address": "MY WALLET",
            // Max Active Jobs
            "maxActiveJobs": 20,
	    // Block rewards go to the configured pool wallet address to later be paid out
            // to miners, except for a percentage that can go to, for examples,
            // pool operator(s) as pool fees or or to donations address. Addresses or hashed
            // public keys can be used. Here is an example of rewards going to the main pool
            // "op"
            "rewardRecipients": [

**Comment by jvxis** [comment]
Hi, @hjonathan2019 I'm using "minimunConfimations": 720

**Reply by hjonathan2019** [reply]
actualy, i have to add those lines manually, becouse my json file didnt have them... but 720 confirmation its almost a day... i would like to reduce to 70 for example (without breaking my json file jajajaja)

**Reply by Konstantin35** [reply]
> actualy, i have to add those lines manually, becouse my json file didnt have them... but 720 confirmation its almost a day... i would like to reduce to 70 for example (without breaking my json file jajajaja)

you probably don't realize that ergo has 720 confirmations to mature a block.

**Reply by hjonathan2019** [reply]
but we can change that value, I understand that the pool has to have a positive balance, by having this positive balance in the wallet, it makes the payment and it is the pool that then waits to collect the block, but the miners were paid before. can be?

**Reply by jvxis** [reply]
Yes, you can do this with you have enough balance in the pool wallet.

**Reply by hjonathan2019** [reply]
ok, soy, back to the start, The configuration that I uploaded, is it correct?

**Reply by jvxis** [reply]
I can't see anything wrong there. Do you have errors?

**Reply by hjonathan2019** [reply]
I haven't tried yet, I wanted you to tell me if it was correct

**Reply by jvxis** [reply]
I'm just a user like you. It seems correct to me.

**Reply by hjonathan2019** [reply]
@oliverw

**Reply by hjonathan2019** [reply]
anyone can confirm???

**Reply by MiningCryptoLive** [reply]
Confirm what? minimum confirmations works? you can put it anywhere in your config. run it and if miningcore runs then you have it correct. If not it will throw an error

**Reply by hjonathan2019** [reply]
this part i want to confirm

"pools": [
// Repeat the following section for multiple coins
{
// DON'T change the id after a production pool has begun collecting shares!
"id": "MYIDHERE",
**"paymentprocessing": {
"minimumConfirmations": 70**
}
"enabled": true,
"coin": "ergo",
// Address to where block rewards are given (pool wallet)
"address": "MY WALLET",
// Max Active Jobs
"maxActiveJobs": 20,
// Block rewards go to the configured pool wallet address to later be paid out
// to miners, except for a percentage that can go to, for examples,
// pool operator(s) as pool fees or or to donations address. Addresses or hashed
// public keys can be used. Here is an example of rewards going to the main pool
// "op"
"rewardRecipients": [

**Reply by oliverw** [reply]
> this part i want to confirm

A bit of a friendly advice. If you need that much handholding to get the basic configuration done, you probably shouldn't run a pool until you got way more experience managing systems in general. 🙄

**Reply by hjonathan2019** [reply]
@oliverw I'm sorry, but I have a disease, dyslexia, and I'm confused by the { } . I do what I can with what I have. I find these things very difficult sometimes.

**Reply by hjonathan2019** [reply]
and one more thing, as an example... what could, for example, a person like Shaquille O'Neal's know about food or other businesses other than the NBA???:

Shaquille O'Neal's franchise ventures:

Papa John's
Auntie Annie's
Car Washes
Five Guys
24-hour Fitness
Krispy Kreme

If he could, why not me?

**Reply by hjonathan2019** [reply]
YOU DREAM. WE BUILD.
Running a Crypto Mining Pool can be a lucrative endeavor, but building a Mining Pool is hard. Attracting miners to a pool is even harder, yet losing them forever is quite easy. This is why it is so important to get it right the first time!

**We’re here to take care of the heavy lifting, while you can focus on your goals!** Your pool will be built – and optionally maintained – by the lead developer of the premier [open source mining pool ](https://github.com/oliverw/miningcore/)solution. Once your pool is operational you won’t be on your own. Support during your critical first month of operation is part of the package. Long-term support contracts can be negotiated individually.

if i buy you the product, i cant run a pool either???

**Reply by hjonathan2019** [reply]
@oliverw Like I told you, I do what I can with what I have, until I can buy the product from you. If I don't get it to work and be profitable now, I'll never be able to buy it

**Reply by oliverw** [reply]
> YOU DREAM. WE BUILD.
> 
> Running a Crypto Mining Pool can be a lucrative endeavor, but building a Mining Pool is hard. Attracting miners to a pool is even harder, yet losing them forever is quite easy. This is why it is so important to get it right the first time!
> 
> 
> 
> **We’re here to take care of the heavy lifting, while you can focus on your goals!** Your pool will be built – and optionally maintained – by the lead developer of the premier [open source mining pool ](https://github.com/oliverw/miningcore/)solution. Once your pool is operational you won’t be on your own. Support during your critical first month of operation is part of the package. Long-term support contracts can be negotiated individually.
> 
> 
> 
> if i buy you the product, i cant run a pool either??? 
> 
> 

Right now we are communicating on GitHub where everything is free. Please don't mix up stuff.

**Reply by hjonathan2019** [reply]
ok, but How many people entered miningcore for being free and then went to pro? github has a lot to do with it. (i think)

**Reply by hjonathan2019** [reply]
well.. i put it here
![image](https://user-images.githubusercontent.com/59586038/197832969-d22732b1-9642-45ab-a7f5-fb56d080ecce.png)

hope it works

**Reply by hjonathan2019** [reply]
> well.. i put it here ![image](https://user-images.githubusercontent.com/59586038/197832969-d22732b1-9642-45ab-a7f5-fb56d080ecce.png)
> 
> hope it works

anyone can confirm this?

**Reply by oliverw** [reply]
That looks fine.

**Reply by hjonathan2019** [reply]
> That looks fine.

awesome @oliverw . thank you very very much  for your time
